src/instana/instrumentation/sanic_inst.py

```diff
@@ -1,144 +1,125 @@
-# (c) Copyright IBM Corp. 2021
-# (c) Copyright Instana Inc. 2021
+# (c) Copyright IBM Corp. 2024
```

Comment:
Why change the Copyright header if this is not a new file?

Reply:
Maybe this happened when I shifted contents from the `sanic_middleware` file I had before auto-instrumenting sanic. Changed it now.